src/remodel_api/remodel.rs

```diff
@@ -637,10 +646,33 @@ impl UserData for Remodel {
         methods.add_function(
             "writeNewPlaceAsset",
             |context, (instance, options): (LuaInstance, Table)| {
-                let name = get_required_option(context, &options, "name")?;
-                let description = get_option(context, &options, "description", String::new())?;
-                let is_public = get_option(context, &options, "isPublic", false)?;
-                let allow_comments = get_option(context, &options, "allowComments", false)?;
+                let name: Option<String> = get_option(context, &options, "name")?;
+                let name = match name {
+                    Some(value) => value,
+                    None => {
+                        return Err(rlua::Error::external(format!(
+                            "The option name must be specified",
+                        )))
+                    }
+                };
+
+                let description = get_option(context, &options, "description")?;
+                let description = match description {
```

Comment:
For all of these, I think you want `.unwrap_or_default()` instead for succinctness.